core/modules/system/system.menu.inc

```diff
@@ -204,7 +207,15 @@ function system_menu_block_build(array $config) {
       $id = backdrop_html_id('menu-toggle-state');
       $data['content']['#wrapper_attributes']['data-menu-toggle-id'] = $id;
       $data['content']['#prefix'] = theme('menu_toggle', array('enabled' => $config['toggle'], 'id' => $id, 'text' => t('Menu')));
-      $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.css';
+      if (config_get('system.core', 'menu_breakpoint') == 'default') {
+        $data['content']['#attached']['css'][] = backdrop_get_path('module', 'system') . '/css/menu-toggle.theme.css';
```

Comment:
Could we move this `['#attached']['css']` to a new entry in `system_library_info()` so that the switching logic based on `config_get('system.core', 'menu_breakpoint')` would go in one function rather than both here and in `hook_library_info()`? Then attach the library instead here with no switching logic:

```
$data['content']['#attached']['library'][] = array('system', 'backdrop.menu-toggle');
```